### ISSUE TITLE
Allow users to upload protein databases (.fasta, .xml)

### ISSUE THREAD

**Issue by zrolfs** [BIRL/PERCEPTRON#8]
The current options available on the web-version are Human, E. coli, and Bovine. If I wanted to study another organism (e.g. yeast) or I have a sample-specific RNA-Seq database, I'm out of luck.

Implementation shouldn't be too difficult, as the nuget package [mzLib ](https://github.com/smith-chem-wisc/mzLib) has open-source code for loading databases.

**Comment by BIRL** [comment]
Yes, we will be happy to do that. Lets setup a meeting, if you prefer?
*----*
*Safee Ullah Chaudhary*, Ph.D. | Assistant Professor,
Biomedical Informatics Research Laboratory (BIRL),
Department of Biology, SBA School of Science and Engineering,
Lahore University of Management Sciences (LUMS), Lahore, Pakistan
Voice: +92 42 3560 8352  | Cell: +92 321 4255171 | Fax: +92 42 3560 8317
Email: ***@***.***, ***@***.***
Web: biolabs.lums.edu.pk/BIRL

* *This email message is for the sole use of the intended recipient(s) and
may contain confidential **information. Any unauthorized review, use,
disclosure or distribution is prohibited. *

**Comment by zrolfs** [comment]
Sure! We can coordinate via e-mail (zrolfs@wisc.edu). I'm busy today after 3:30 PM ET, but otherwise available.
